src/esi/ExpatParser.cc

```diff
@@ -12,8 +12,28 @@
 
 #if USE_SQUID_ESI && HAVE_LIBEXPAT
 
+#include "base/RunnersRegistry.h"
 #include "esi/ExpatParser.h"
 
+static ESIParser::Register *prExpat = nullptr;
+
+class EsiExpatRr : public RegisteredRunner
```

Comment:
Please place this new class inside the Esi namespace, adjusting its name accordingly. I hope that moving the corresponding RunnerRegistrationEntry() declaration/macro into the same Esi namespace will make that namespace-unaware macro work correctly.

Reply:
Done in d28e46a